datajoint/expression.py

```diff
@@ -442,19 +454,30 @@ def __bool__(self):
 
     def __contains__(self, item):
         """
-        returns True if item is found in the .
+        returns True if a restriction results with any records e.g. restriction in q1.
         :param item: any restriction
         (item in query_expression) is equivalent to bool(query_expression & item) but may be
         executed more efficiently.
         """
         return bool(self & item)  # May be optimized e.g. using an EXISTS query
 
     def __iter__(self):
+        """
+        returns an iterator-compatible QueryExpression object e.g. iter(q1).
+
+        :param self: iterator-compatible QueryExpression object
+        """
         self._iter_only_key = all(v.in_key for v in self.heading.attributes.values())
         self._iter_keys = self.fetch('KEY')
         return self
 
     def __next__(self):
+        """
+        returns the next record on an iterator-compatible QueryExpression object
+            e.g. next(q1).
+
+        :param self: fetch1 record
```

Comment:
Here self is a QueryExpression that has been primed with `iter` to yield dicts containing the primary key values.